language-design.md

```diff
@@ -38,7 +69,14 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well-designed language is extendible, has a relatively small core vocabulary, is easy to learn (though not necessarily for everyone),
+is easy for the user to read and write, and has good documentation.
```

Comment:
Perhaps documentation (like many other types of written content) can be optimized for different audiences. For example, I now find [this](http://www.cplusplus.com/reference/set/set/) explanation of C++ [ordered] sets to be very helpful, but earlier on in my experience with C++ I would have preferred [this](https://www.topcoder.com/community/data-science/data-science-tutorials/power-up-c-with-the-standard-template-library-part-1/#set)